packages/bls12-381-gen/bls12-381-gen.0.4.1/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+synopsis: "Functors to generate BLS12-381 primitives based on stubs"
+description: "Functors to generate BLS12-381 primitives based on stubs"
+maintainer: "Danny Willems <be.danny.willems@gmail.com>"
+authors: "Danny Willems <be.danny.willems@gmail.com>"
+license: "MIT"
+homepage: "https://gitlab.com/dannywillems/ocaml-bls12-381"
+bug-reports: "https://gitlab.com/dannywillems/ocaml-bls12-381/issues"
+depends: [
+  "ocaml" {>= "4.08" & < "4.12"}
+  "dune" {>= "2.7"}
+  "ff-sig" {>= "0.6.1" & < "0.7.0"}
+  "zarith" {>= "1.10" & < "2.0"}
+  "bisect_ppx" {with-test & >= "2.5" & < "2.6.0"}
```

Comment:
Same question with that actually. I believe this is just a bug fixed by https://github.com/ocaml/opam-repository/pull/18141

Reply:
Yes I do confirm `< "2.6.0"` is just a temporary workaround because the tests were failing in the CI here.